test-site-node/index.js

```diff
@@ -1,7 +1,7 @@
 const { provideHeadless } = require('@yext/search-headless');
 
 const answers = provideHeadless({
-  apiKey: '2d8c550071a64ea23e263118a2b0680b',
```

Comment:
can we modify this to use an environment variable, e.g. `process.env.API_KEY` that should allow us to do `API_KEY=realkey npm run start`

Reply:
Updated!